contracts/lease/src/contract/state/opened/active.rs

```diff
@@ -67,22 +70,25 @@ impl Active {
     }
 
     fn try_repay(self, deps: Deps<'_>, env: Env, info: MessageInfo) -> ContractResult<Response> {
-        let payment = bank::may_received::<PaymentGroup, _>(
+        let payment = never!(bank::may_received::<PaymentGroup, _>(
```

Comment:
we do not prefer macro-ses when regular functions would work. look at dex::never